docs/migrations/chain-upgrade-guide-040.md

```diff
@@ -0,0 +1,228 @@
+# Upgrading a live chain from 039 to 040
+
+### Risks
+As a validator performing the upgrade procedure on your consensus nodes carries a heightened risk of double-signing and
+being slashed. The most important piece of this procedure is verifying your software version and genesis file hash before
+starting your validator and signing.
+
+The riskiest thing a validator can do is discover that they made a mistake and repeat the upgrade procedure again during
+the network startup. If you discover a mistake in the process, the best thing to do is wait for the network to start
+before correcting it. If the network is halted and you have started with a different genesis file than the expected one,
+seek advice from the validator community.
+
+### Recovery
+* Prior to exporting the state, the validators are encouraged to take a full data snapshot at exported height. Exported
+height will be determined by a governance proposal. Data backup is usually done by copying daemon home directory,
+ex: `~/.gaiad`
```

Comment:
I think it would be nice to add a note: we use "gaiad" as our app, be sure to replace with the name of your own binary...

Reply:
Thanks @amaurymartiny, updated it and also changed it to `simd`